utils/src/main/java/io/atomix/utils/serializer/Namespace.java

```diff
@@ -358,11 +358,9 @@ public Namespace populate(int instances) {
    * @param buffer to write to
    */
   public void serialize(final Object obj, final ByteBuffer buffer) {
-    ByteBufferOutput out = new ByteBufferOutput(buffer);
     Kryo kryo = borrow();
-    try {
+    try (ByteBufferOutput out = new ByteBufferOutput(buffer)) {
```

Comment:
Oh good that they are closed now😮

Reply:
Haha :see_no_evil: 

I saw them and was wondering, wait, isn't this a problem? :thinking:

Reply:
No wonder that Atomix uses so much mem 😅🤣